dotcom-rendering/src/layouts/LiveLayout.tsx

```diff
@@ -1170,6 +1170,7 @@ export const LiveLayout = (props: WebProps | AppsProps) => {
 											margin-left: 20px;
 											margin-right: -20px;
 											display: none;
+											padding-bottom: 255px;
```

Comment:
Nice. I like this solution much better than the one I suggested 👍 

I have tested locally and it works well. It may be slightly more accurate to move this line of css to the `${from.wide}` breakpoint, as the right column only appears on liveblogs on wide (>1300px) screen sizes

Reply:
Thank you Dom! Yes it makes sense to have it in wide breakpoint if `right-column` doesn't appear before that breakpoint. I made that change.